content/themes/semantic-ui/amazon/index.slim

```diff
@@ -9,8 +9,9 @@
       a.item data-tab="views" Views
       a.item data-tab="modules" Modules
 
-.ui.demo.container
+.ui.demo.container.basic.segment
```

Comment:
Could you explain why would you like to make it a `.container.basic.segment`?

I checked http://semantic-ui.com/kitchen-sink.html page and found it is a `.basic.container`,